TRANSLATION.md

```diff
@@ -0,0 +1,77 @@
+# reactjs.org saytining O'zbek tilidagi tarjimasi
```

Comment:
How this file end up here again? Try to sync your fork https://help.github.com/articles/syncing-a-fork/

Reply:
hm, i synced master only